internal/pkg/gateway/server/api.go

```diff
@@ -79,10 +85,33 @@ func (gs *Server) Endorse(ctx context.Context, proposedTransaction *pb.ProposedT
 	return preparedTxn, nil
 }
 
+//go:generate counterfeiter -o mocks/submitserver.go --fake-name SubmitServer . submitServer
+type submitServer interface {
+	pb.Gateway_SubmitServer
+}
+
```

Comment:
looks like this belongs in the test files.

Reply:
moved